public/index.html

```diff
@@ -35,6 +35,7 @@
     <title>Falling Fruit</title>
   </head>
   <body>
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyCc8tJxECaXE2OX6MbByAKSOPs6XGuTvhg&libraries=places"></script>
```

Comment:
We should never include API keys in source code. https://developers.google.com/maps/api-key-best-practices

Can you look into using environment variables here?

Reply:
Also, can you move this script tag to the head, add a comment for what it is, and use asynchronous script loading? https://github.com/wellyshen/use-places-autocomplete#load-the-library